example/goodrx/index.ts

```diff
@@ -0,0 +1,31 @@
+import { start } from "../../src/http";
+import { Config, Http, CDSHooks } from "../../src"
+
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
Only thing preventing building is I think a merge conflict, but other than that all looks good!

Reply:
Yikes, awkward